source/ota.c

```diff
@@ -2282,8 +2493,8 @@ static IngestResult_t decodeAndStoreDataBlock( OtaFileContext_t * pFileContext,
         }
         else
         {
-            *uBlockIndex = ( uint32_t ) sBlockIndex;
-            *uBlockSize = ( uint32_t ) sBlockSize;
+            *blockIndex = ( uint32_t ) sBlockIndex;
+            *blockSize = ( uint32_t ) sBlockSize;
```

Comment:
See above comments about adding a "p" depicting a pointer.

Reply:
Thanks updated in https://github.com/aws/ota-for-aws-iot-embedded-sdk/pull/116/commits/3e5b1d96211d212a79fd5aa4a1bd33a51add1e85